SpringProject/src/it/java/com/gradproject2019/conferences/ConferenceControllerIT.java

```diff
@@ -4,209 +4,179 @@
         Conference retrievedConference = conferenceRepository.findAll().get(0);
 
-        //Then
         Assert.assertEquals(200, response.getStatusCodeValue());
         Assert.assertEquals(conferenceRequestDto.getName(), retrievedConference.getName());
     }
 
     @Test
     public void shouldReturn400WhenAnyFieldNull() throws URISyntaxException {
-        //given
-        URI uri = new URI(baseUrl);
-        HttpEntity<ConferenceRequestDto> request = new HttpEntity<>(createRequestDto( null, null, null, null, null));
+        URI uri = new URI(baseUri);
+        ConferenceRequestDto request = createRequestDto(null, null, null, null, null);
 
-        //when
         ResponseEntity<String> response = postConference(uri, request);
 
-        //Then
-        Assert.assertEquals(400,response.getStatusCodeValue());
+        Assert.assertEquals(400, response.getStatusCodeValue());
     }
 
     @Test
     public void shouldReturn400WhenConferenceInPast() throws URISyntaxException {
-        //given
-        URI uri = new URI(baseUrl);
-        HttpEntity<ConferenceRequestDto> request = new HttpEntity<>(createRequestDto("Grace's conference", Instant.parse("2018-12-30T19:34:50.63Z"), "Leicester", "All about Grace's fabulous and extra house", "grace"));
+        URI uri = new URI(baseUri);
+        ConferenceRequestDto request = createRequestDto("Grace's conference", Instant.parse("2018-12-30T19:34:50.63Z"), "Leicester", "All about Grace's fabulous and extra house", "grace");
 
-        //when
         ResponseEntity<String> response = postConference(uri, request);
 
-        //Then
-        Assert.assertEquals(400,response.getStatusCodeValue());
+        Assert.assertEquals(400, response.getStatusCodeValue());
     }
 
     @Test
     public void shouldReturn404WhenConferenceToBeDeletedNotFound() throws URISyntaxException {
-        //given
-        URI uri = new URI(baseUrl + "/1000000000");
+        URI uri = new URI(baseUri + "/1000000000");
 
-        //when
         ResponseEntity<String> response = deleteConference(uri);
 
-        //then
-        Assert.assertEquals(404,response.getStatusCodeValue());
+        Assert.assertEquals(404, response.getStatusCodeValue());
     }
 
     @Test
     public void shouldReturn204AndDeleteConference() throws URISyntaxException {
-        //given
         Conference savedConference = conferenceRepository.saveAndFlush(conference);
-        URI uri = new URI(baseUrl + "/" + savedConference.getId());
+        URI uri = new URI(baseUri + "/" + savedConference.getId());
 
-        //when
         ResponseEntity<String> response = deleteConference(uri);
 
-        //then
-        Assert.assertEquals(204,response.getStatusCodeValue());
+        Assert.assertEquals(204, response.getStatusCodeValue());
         Assert.assertFalse(conferenceRepository.existsById(savedConference.getId()));
     }
 
+    @Test
+    public void shouldReturn401WhenUnauthorised() throws URISyntaxException {
```

Comment:
Rename to something like shouldReturn401WhenUnauthorizedDelete to avoid a clash with shouldReturn401WhenUnauthorizedEdit.